opentext-page.html

```diff
@@ -0,0 +1,482 @@
+                    <a href="#Protect">Protect patient data</a>                
+                </div>
+            </div>
+        </div>
+        <div class="client">
+            <div class="row">
+                <div class="col-4">
+                    <a href="#Achieve">Achieve website management ROI</a>               
+                </div>
+            </div>
+        </div>
+        <div class="client">
+            <div class="row">
+                <div class="col-4">
+                    <a href="#L'oreal">L'oreal logo</a>
+                </div>
+            </div>
+        </div>
+        <div class="client">
+            <div class="row">
+                <div class="col-4">
+                    <a href="#Sprint">Sprint logo</a>
+                </div>
+            </div>
+        </div>
+        <div class="client">
+            <div class="row">
+                <div class="col-4">
+                    <a href="#Philips">Philips logo</a>                
+                </div>
+            </div>
+        </div>
+        <div class="client">
+            <div class="row">
+                <div class="col-4">
+                    <a href="#Vodafone">Vodafone logo</a>                               
+                </div>
+            </div>
+            <button type="submit">Read more Sucess stories</button>
+        </div>
+        <div class="blog">
+            <div class="row">
+                <div class="column">
+                    <a href="#News">Latest news and insights</a>
+                        <!-- no borders with an arrow -->
+                        <button type="submit">See All</button>
+                    </a>
+                </div>
+            </div>
+        </div>
+        <div class="blog">
+            <div class="row">
+                <div class="column">
+                    <!--has arrow --> 
+                    <a href="#Solving">Soliving the review team conundrum</a>                           
+                </div>
+            </div>
+        </div>
+        <div class="blog">
+            <div class="row">
+                <div class="column">
+                    <!--has arrow --> 
+                    <a href="#Technology">Top 3 technology trends for Healthcare in 2021</a>                             
+                </div>
+            </div>
+        </div>
+        <div class="blog">
+            <div class="row">
+                <div class="column">
+                    <!--has arrow --> 
+                    <a href="#Content">We need to talk about content</a>                               
+                </div>
+            </div>
+        </div>
+        <div class="blog">
+            <div class="row">
+                <div class="column">
+                    <!--has arrow --> 
+                    <a href="#Name">What is in a name?</a>                              
+                </div>
+            </div>
+        </div>
+        <footer class="footer">
```

Comment:
The `<footer>` tag is misused.
It's not a tag this is used over and over for footer columns.
Instead, this is the format.  Please re-do this entire footer section.

```
<footer class="container">
    <div class="row">
       <div class="column">
           ... links and stuff here
       </div>
       <div class="column">
           ... links and stuff here
       </div>
       <div class="column">
           ... links and stuff here
       </div>
   </div>
</footer>
```